aten/src/ATen/native/native_functions.yaml

```diff
@@ -3355,6 +3355,8 @@
 
 - func: _sparse_coo_tensor_unsafe(Tensor indices, Tensor values, int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
+- func: _validate_sparse_coo_tensor_args(Tensor indices, Tensor values, int[] size) -> bool
```

Comment:
This returns `bool` simply because I didn't know how to add a function that returns `void`/`None`. (The only purpose of this function is to throw if something is wrong)

Reply:
Could you try `void` and `NoneType`?

Reply:
I tried both. In both cases I get:

```
  File "/home/wbaranowski/git/Quansight/pytorch/tools/pyi/gen_pyi.py", line 137, in type_to_python                                                                        
    }[typename]                                                                                                        
KeyError: 'NoneType'
```

```
  File "/home/wbaranowski/git/Quansight/pytorch/tools/autograd/gen_python_functions.py", line 382, in get_simple_return_type
    raise RuntimeError(declaration['name'] + " returns unsupported type " + simple_return_type)        
RuntimeError: _validate_sparse_coo_tensor_args returns unsupported type NoneType
```

during the build.